backend/app/views/user_mailer/welcome_email.html.erb

```diff
@@ -0,0 +1 @@
+sdfsdf
```

Comment:
What is it? And why is it here?

Reply:
> 

It is good. But will be better if you change that text like "randomtxttomail" or "random_txt_to_mail"
@SilaEwGenii

Reply:
I don't think this is a breaking change, as this text will change in the welcome email with text that already makes sense. The path to the file in full display the meaning of the given file. @LorDEvilDead